docs/test_plan.rst

```diff
@@ -0,0 +1,155 @@
+LORE Test Plan
+==============
+
+LORE is known to render differently in Firefox and Chrome, with Chrome
+producing less satisfactory results. So testing on Chrome is
+recommended.
+
+#. .. rubric:: As Administrator...
+      :name: as-administrator...
+
+   1. .. rubric:: Repositories
+         :name: repositories
+
++-----------------------+----------+-------------------------------------------------------------------------+
+| Action                | Result   | Notes                                                                   |
++-----------------------+----------+-------------------------------------------------------------------------+
+| Create a repository   |          | You may only create a repository if you are a member of the '' group.   |
++-----------------------+----------+-------------------------------------------------------------------------+
++-----------------------+----------+-------------------------------------------------------------------------+
+
+Courses
+-------
+
+Expect a large course import to be slow, anywhere from several minutes
+to several hours.
+
++-------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+| Action            | Result                                                                                          | Notes                                                                                                                                         |
++-------------------+-------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+| Import a course   | Course title appears in 'Course' facet list with the total number of LRs following the title.   | Since many LORE issues are related to large size and "creative" content, import large and potentially problematic courses such as 8.01x ().   |
```

Comment:
Can we include the expected number of resources for a specific course (specified by URL)? It would be nice if it were an open course, but I could live with a closed one.